app/src/main/java/com/arygm/quickfix/ui/uiMode/appContentUI/workerMode/home/EarningsWidget.kt

```diff
@@ -0,0 +1,162 @@
+package com.arygm.quickfix.ui.uiMode.appContentUI.workerMode.home
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme.colorScheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.arygm.quickfix.ui.theme.poppinsTypography
+import java.text.DecimalFormat
+
+@Composable
+fun EarningsWidget(
+    personalBalance: Double,
+    earningsThisMonth: Double,
+    avgSellingPrice: Double,
+    activeOrders: Int,
+    activeOrderValue: Double,
+) {
+  BoxWithConstraints {
+    val horizontalSpacing = maxWidth * 0.025f // 2.5% of the available width for spacing
+    val currencyFormat = DecimalFormat("######.##CHF")
+
+    Column(
+        modifier =
+            Modifier.fillMaxWidth()
+                .padding(horizontalSpacing)
+                .shadow(5.dp, RoundedCornerShape(20.dp))
+                .background(colorScheme.surface, RoundedCornerShape(12.dp))
+                .testTag("EarningsWidget"),
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.Start) {
+          // Header Row
+          Row(
+              modifier =
+                  Modifier.fillMaxWidth()
+                      .padding(horizontal = 12.dp)
+                      .padding(top = 12.dp)
```

Comment:
Same here, you can use `maxHeight` to remove the use of absolute values.
```suggestion
                      .padding(top = 12.dp * maxHeight.value)
```